packages/api-server/src/handlers/op.ts

```diff
@@ -17,16 +25,15 @@ const logger = new ObjLogger("API");
  */
 export function createOperationHandler(router: SequentialCeroRouter): APIRoute["op"] {
     /**
-     * Tries to parse data from the request body.
-     *
+     * Checking content-type and getting encoding from request
      * @param {IncomingMessage} req Request object.
-     * @returns JSON object.
+     * @param {boolean} rawBody Flag if the body will be parsed.
+     * @returns BufferEncoding string
      */
-    const getData = async (req: IncomingMessage): Promise<object|undefined> => {
-        if (!req.headers["content-type"])
-            throw new CeroError("ERR_INVALID_CONTENT_TYPE");
+    const getEncoding = (req: IncomingMessage, rawBody: boolean = false): BufferEncoding => {
```

Comment:
I'd rather we did this like this:

```suggestion
    const getEncoding = (req: IncomingMessage, { rawBody: boolean = false }): BufferEncoding => {
```

WDYT?